tests/gold/concatV.h

```diff
@@ -13,6 +13,18 @@
 #include <complex.h>
 
 
-void concatV(struct array * v1, struct array * * out);
+struct awl_signedS32
+{
+  int32_t * buffer;
```

Comment:
Calling it "data" would be shorter.

Reply:
So would calling it "x" or "foo". But letting it keep its name "buffer" gives a shorter diff.